### ISSUE TITLE
Styling: use css prop from styled-components instead of style

### ISSUE THREAD

**Issue by Evalir** [pokt-network/wpokt-app#3]

**Comment by ECWireless** [comment]
I think I'm still a tad confused about what an example of this would look like.

Do you mean doing something like this? https://github.com/quadnetwork/aragon-ui/blob/master/src/components/Button/Button.js#L29

**Comment by Evalir** [comment]
So, I actually put a comment in #5 talking about this, but let me explain myself better. :)

Right now, we're making components like this:

```
const StyledWrapper = styled.div`
  width: 100%;
`

function Component() {
  return (
    <Wrapper>
      {/* arbitrary style */}
      <span style={{ display: 'inline-block' }}>hi</span>
    </Wrapper>
  )
}
```

As we're using styled-components, we have access to its [css prop](https://styled-components.com/docs/api#css-prop), which lets us inject individual styles without having to write them by objects. As we're already all-in on styled-components, for the sake of consistency let's stick to using it on these cases, like so:

```
const StyledWrapper = styled.div`
  width: 100%;
`

function Component() {
  return (
    <Wrapper>
      {/* arbitrary style */}
      <span css=`display: inline-block`>hi</span>
    </Wrapper>
  )
}
```

We can make use of the prop by:

- If we're importing styled in the same file, do: `import styled from 'styled-components/macro'`
- If we're not importing styled and just need the CSS prop, do: `import 'styled-components/macro'`

**Comment by ECWireless** [comment]
Closed with #5
